src/webvr-manager.js

```diff
@@ -271,6 +273,15 @@ WebVRManager.prototype.onOrientationChange_ = function(e) {
   this.updateRotateInstructions_();
 };
 
+WebVRManager.prototype.onResizeChange_ = function() {
```

Comment:
Not sure what other way we can detect for orientation changes without orientationchange event.  Open to suggestions if you have them :)

Pushing to get the event added to Firefox Android so that we won't have to work around this.